docs/references.rst

```diff
@@ -4,19 +4,56 @@ Ubuntu Pro Client reference
 ***************************
 
 Our reference section contains technical information for the Ubuntu Pro Client.
-This includes details of the network requirements, API definitions, and other
-related tools.
 
-Reference
-=========
+System and other requirements
+=============================
+
+The Ubuntu Pro Client is lightweight and has very few requirements. 
```

Comment:
If this is true, does this need to be a standalone sub-section?

Reply:
Probably not, but I wasn't sure where else to put it :upside_down_face: in the absence of a better place I'll leave it in situ for now